spec/services/invitations/send_sms_spec.rb

```diff
@@ -23,26 +23,27 @@
       region: "Auvergne-Rhône-Alpes"
     )
   end
+  let!(:configuration) { create(:configuration, context: "rsa_orientation") }
+  let!(:organisation) { create(:organisation, configurations: [configuration], department: department) }
 
   let!(:invitation) do
     create(
       :invitation,
       applicant: applicant, department: department, token: "123", help_phone_number: help_phone_number,
-      number_of_days_to_accept_invitation: 9,
+      number_of_days_to_accept_invitation: 9, organisations: [organisation],
       link: "https://www.rdv-solidarites.fr/lieux?invitation_token=123", format: "sms", rdv_context: rdv_context
     )
   end
 
   let!(:rdv_context) { build(:rdv_context, context: "rsa_orientation") }
+  let!(:content) do
+    "Monsieur John DOE,\nVous êtes bénéficiaire du RSA et vous devez vous présenter à un rendez-vous "\
+      "d'orientation. Pour choisir la date et l'horaire de votre premier RDV, cliquez sur le lien suivant "\
+      "dans les 9 jours: http://www.rdv-insertion.fr/invitations/redirect?token=123\n"\
+      "Ce rendez-vous est obligatoire. En cas de problème technique, contactez le 0147200001."
+  end
 
   describe "#call" do
-    let!(:content) do
-      "Monsieur John DOE,\nVous êtes bénéficiaire du RSA et vous devez vous présenter à un rendez-vous "\
-        "d'orientation. Pour choisir la date et l'horaire de votre premier RDV, cliquez sur le lien suivant "\
-        "dans les 9 jours: http://www.rdv-insertion.fr/invitations/redirect?token=123\n"\
-        "Ce rendez-vous est obligatoire. En cas de problème technique, contactez le 0147200001."
-    end
-
```

Comment:
A quoi sert ce changement ?

Reply:
à ne pas redéfinir `content` pour le `context "when the sms sender name is defined in organisation configuration"` à la ligne 163